### DISCUSSION TITLE
Numbers appearing after suggestions?

### DISCUSSION THREAD

**Discussion by corey1031d** [florisboard/florisboard#2064] [Q&A]
Hello, im new to Floris and loving it, but i seem to be having an issue where numbers appear after every suggestion. How do i get rid of this?
![Screenshot_20220908-193541](https://user-images.githubusercontent.com/33399008/189244264-ea5bab80-5d13-4758-b3ce-9ef75f61ba91.png)

**Comment by Beach77** [comment]
![20220909_115340.jpg](https://user-images.githubusercontent.com/101499628/189323740-85843163-6cd7-4069-ae5a-d30a76cef49f.jpg)
